### ISSUE TITLE
PHP 7.3 support

### ISSUE THREAD

**Issue by DmitriyArbuzov** [splashlab/magento-2-cors-requests#17]
Hi,

Are you going to release the next version with PHP 7.3 support?

Thanks!

**Comment by evs-xsarus** [comment]
Also need one. @DmitriyArbuzov for now, you can use this repository:

```
        "4": {
            "type": "vcs",
            "url": "https://github.com/bitbull-team/magento-2-cors-requests"
        },
```

```composer require splashlab/magento-2-cors-requests:"dev-master#0cef952```

**Comment by thaddeusmt** [comment]
Sorry this languished for so long! I merged in the pull requests to support PHP 7.3 and tagged a new release [100.0.4](https://github.com/splashlab/magento-2-cors-requests/releases/tag/100.0.4)
